examples/flexvolume/example_bi.yaml

```diff
@@ -0,0 +1,26 @@
+apiVersion: v1
+kind: Pod
+metadata:
+  name: flexvol-baseimage
+  namespace: default
+spec:
+  containers:
+  - name: flexvol-baseimage
+    image: nginx:stable-alpine
+    imagePullPolicy: IfNotPresent
+    volumeMounts:
+    - name: flexvol
+      mountPath: /usr/share/nginx/html
+    ports:
+    - containerPort: 80
+  volumes:
+  - name: flexvol
+    flexVolume:
+      driver: "rancher.io/longhorn"
+      # fsType: "iso9660"
+      options:
+        size: "16Mi"
```

Comment:
You've updated the test base image? It's better for the size to be different from the test image size, so we can catch unexpected errors.

Reply:
Probably not 2G, because we're going run whole disk checksum for it often. 

32MB should be good enough.